laravel/app/Http/Controllers/UserController.php

```diff
@@ -13,4 +17,47 @@ public function user_logout() {
     public function __construct(){
         $this->middleware('auth');
     }
+
+    public function show()
+    {
+        if (Auth::check() === false )
+        {
+            return redirect('top');
+        }
+
+        $user = Auth::user();
+
+        //default値の画像ディレクトリが違うのでパスを条件分岐しておく
+        if ($user->image == "no_image.png")
+        {
+            $user_image_path =   asset("/SystemImage/no_image.png");
+        }else{
+            $user_image_path =   asset("/UserImage/$user->image");
+        }
+
+        $posts = User::find(Auth::id())->get_post()->where('user_id',Auth::id())->get();
+
+        return view('profile',compact('user','posts','user_image_path'));
+    }
+
+    //ユーザー情報変更(画像・ユーザー名・自己紹介文)
+    public function store(ValidateController $request)
+    {
+        $user = Auth::user();
+        $user->name = $request['name'];
+        $user->comment = $request['comment'];
+
+        //formから画像ファイルの実態を受け取っていたら画像ファイル名取得し、保存する
+        if (isset($request['image']) === true)
```

Comment:
この様に書いたほうがより明確ではあるのですが、下記のように書いたほうがシンプルかもしれません。
```suggestion
        if (isset($request['image']))
```